.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.3.4
```

Comment:
Why was this committed out of curiosity? Wondering if it's necessary!

Reply:
Oh sorry, this is from [asdf](https://github.com/asdf-vm/asdf), a version manager for some programming languages. I use this tool instead of rbenv/rvm and had to set it in order to use that ruby version.

Will delete it. :+1:

Reply:
We could commit it if you think it will be helpful for others!

Reply:
I was looking at the asdf repo, it seems that I forgot to set it up properly (it should check `.ruby-version` and `.node-version` in case `.tools-version` is absent. :bowing_man: